docs/standard/serialization/snippets/system-text-json-how-to/csharp/DeserializeRequiredProperty.cs

```diff
@@ -1,5 +1,4 @@
-﻿using System;
-using System.Text.Json;
+﻿using System.Text.Json;
```

Comment:
Not necessary, but since all of these samples use `System.Text.Json`, you could consider adding it to the global usings.

Reply:
That's a good idea. I'll do that for the next batches of snippet files but will keep the STJ `using` in the files that have the whole file including `using`s displayed in the docs.